### PR TITLE
Added ZSH environment config

### PR DESCRIPTION
**Pull request by mmhj**
Added the following ZSH environment config files:

.zshenv
.zprofile
.zlogin
.zlogout

.zshrc is already included in '*rc'.

### CODE REVIEW

File: LS_COLORS [hunk 1]
```diff
@@ -529,6 +534,8 @@ STICKY_OTHER_WRITABLE 48;5;235;38;5;139;3
 .s7z                  38;5;40
 .sz                   38;5;40
 .tar                  38;5;40
+.tar.lzma             38;5;40
```
Comment (rpdelaney) [inline LS_COLORS:537]: It would probably be better to change `.lzma` to `*.lzma`. Maybe it makes sense to do that with all the archive types actually...

Comment (mmhj) [reply LS_COLORS:537]: I agree, that is probably the best way to do it, but that would require changing all of the archive types.

I'll have a look at it later when I have some free time.

Comment (mmhj) [reply LS_COLORS]: I implemented the changes. 

I also cleaned up the entire document so it looks cleaner. 
https://github.com/trapd00r/LS_COLORS/pull/178/commits/d20f554612a21f18623a00a3a024e7ee225678ed

Comment (rpdelaney) [reply LS_COLORS]: This is great! Thank you!

**Pull request merged**